daemon/internal/newrelic/processor_test.go

```diff
@@ -19,9 +19,11 @@ import (
 	"github.com/newrelic/newrelic-php-agent/daemon/internal/newrelic/utilization"
```

Comment:
Formatting (for this whole file)- 
there was initially a change made to this file that was committed and reverted, but the autoformatter changes persisted